exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/transactions/experimental/Suspended.kt

```diff
@@ -53,26 +54,40 @@ suspend fun <T> newSuspendedTransaction(
     context: CoroutineContext? = null,
     db: Database? = null,
     transactionIsolation: Int? = null,
+    repetitionAttempts: Int = 0,
```

Comment:
If they are moved into the `Transaction` class, I'm not sure how to then access them, since this function always create a new `Transaction` in `TransactionScope` to execute the block statements. The function can access the `db.config` defaults but not any outer `Transaction` details.